base/options/options.html

```diff
@@ -291,4 +291,4 @@
 			</div>
 		</div>
 	</body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
even i get the last line new line char change when I use VIM. Any simple suggestion on how to deal with it?